src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/kafka/KafkaOppfolgingstilfellePerson.kt

```diff
@@ -24,29 +24,39 @@ data class KafkaOppfolgingstilfelle(
 )
 
 fun KafkaOppfolgingstilfellePerson.toOppfolgingstilfelle(
-    latestTilfelle: KafkaOppfolgingstilfelle,
+    selectedTilfelle: KafkaOppfolgingstilfelle,
 ) = Oppfolgingstilfelle(
     personIdent = PersonIdentNumber(this.personIdentNumber),
-    tilfelleStart = latestTilfelle.start,
-    tilfelleEnd = latestTilfelle.end,
-    arbeidstakerAtTilfelleEnd = latestTilfelle.arbeidstakerAtTilfelleEnd,
-    virksomhetsnummerList = latestTilfelle.virksomhetsnummerList.map { Virksomhetsnummer(it) },
+    tilfelleStart = selectedTilfelle.start,
+    tilfelleEnd = selectedTilfelle.end,
+    arbeidstakerAtTilfelleEnd = selectedTilfelle.arbeidstakerAtTilfelleEnd,
+    virksomhetsnummerList = selectedTilfelle.virksomhetsnummerList.map { Virksomhetsnummer(it) },
 )
 
 fun KafkaOppfolgingstilfellePerson.toLatestOppfolgingstilfelle(): Oppfolgingstilfelle? =
-    this.oppfolgingstilfellerWithoutFutureTilfeller().maxByOrNull {
+    this.oppfolgingstilfelleList.maxByOrNull {
         it.start
     }?.let { latestKafkaOppfolgingstilfelle ->
         if (latestKafkaOppfolgingstilfelle.arbeidstakerAtTilfelleEnd) {
             this.toOppfolgingstilfelle(
-                latestTilfelle = latestKafkaOppfolgingstilfelle
+                selectedTilfelle = latestKafkaOppfolgingstilfelle
             )
         } else {
             null
         }
     }
 
-fun KafkaOppfolgingstilfellePerson.oppfolgingstilfellerWithoutFutureTilfeller() =
-    this.oppfolgingstilfelleList.filter {
-        it.start.isBefore(tomorrow())
+fun KafkaOppfolgingstilfellePerson.toCurrentOppfolgingstilfelle(): Oppfolgingstilfelle? {
+    val today = LocalDate.now()
+    return this.oppfolgingstilfelleList.filter {
```

Comment:
Kan bruke `firstOrNull()` med predicate i stedet for først `.filter()` og så `.firstOrNull()` 🤓

Reply:
He, he - tror du påpekte det samme i en annen PR i går.. Skal forsøke å huske det framover 😎

Reply:
Jeg får det opp som forslag i IntelliJ 😎 Får du ikke?

Reply:
Geir bruker light mode, og da blir IntelliJ litt dårligere :trollface: